symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -40,19 +43,19 @@ public SymphonyBdk(BdkConfig config) throws AuthInitializationException, AuthUna
     this.oboAuthenticator = authenticatorFactory.getOboAuthenticator();
   }
 
-  public V4MessageService messages() throws AuthInitializationException {
-    return new V4MessageService(this.apiClientFactory.getAgentClient(), this.botSession);
+  public MessageService messages() {
+    return new MessageService(new MessagesApi(this.apiClientFactory.getAgentClient()), this.botSession);
   }
 
-  public V4MessageService messages(Obo.Handle oboHandle)
-      throws AuthUnauthorizedException, AuthInitializationException {
+  public MessageService messages(Obo.Handle oboHandle)
+      throws AuthUnauthorizedException, ApiClientInitializationException {
     AuthSession oboSession;
     if (oboHandle.hasUsername()) {
       oboSession = this.oboAuthenticator.authenticateByUsername(oboHandle.getUsername());
     } else {
       oboSession = this.oboAuthenticator.authenticateByUserId(oboHandle.getUserId());
     }
-    return new V4MessageService(this.apiClientFactory.getAgentClient(), oboSession);
+    return new MessageService(new MessagesApi(this.apiClientFactory.getAgentClient()), oboSession);
```

Comment:
If we made this change, should we apply it to DatafeedService as well? 
```
if (DatafeedVersion.of(this.config.getDatafeed().getVersion()) == DatafeedVersion.V2) {
      return new DatafeedServiceV2(new DatafeedApi(this.apiClientFactory.getAgentClient()), this.botSession, this.config);
    }
    return new DatafeedServiceV1(new DatafeedApi(this.apiClientFactory.getAgentClient()), this.botSession, this.config);
```

Reply:
Done 👍